README.md

```diff
@@ -1,50 +1,63 @@
 
-# Temporary WoT TD Tooling
+# WoTIS - Web of Things Integrated Schemas
 
-The aim of this repository is to simplify the current WoT TD tooling for generating the specification. WoT TD Tooling is a python project to automate the generation of
- 1) WoT resources: [SHACL Shapes](https://www.w3.org/TR/shacl/), [JSON Schema](https://json-schema.org/specification), [JSON-LD context](https://www.w3.org/TR/json-ld11/), and [RDF](https://www.w3.org/TR/rdf11-concepts/)
+The aim of this repository is to simplify the current tooling required for generating the WoT Thing Description (TD) specification and related resources. 
+WoTIS toolchain is a python-based project designed to automate the generation of
+ 1) WoT resources: [SHACL Shapes](https://www.w3.org/TR/shacl/), [JSON Schema](https://json-schema.org/specification), [JSON-LD context](https://www.w3.org/TR/json-ld11/), [RDF](https://www.w3.org/TR/rdf11-concepts/) and visual figures
  2) Documentation: TD specification and ontology specifications
 
-Here is an overview of the process:
+This project leverages [LinkML](https://linkml.io/linkml/) for modelling the [Web of Things Thing Description](https://www.w3.org/TR/wot-thing-description11/) information model.
 
-<img title="WoT Toolchain Overview" src="images/toolchain.svg">
+## Process Overview
 
-This project leverages [LinkML](https://linkml.io/linkml/) for modelling the [Web of Things Thing Description](https://www.w3.org/TR/wot-thing-description11/) information model.
+Below is a simplified overview of the process:
 
-## Prerequisites
+Step 1: Generate WoT resources using WOTIS
+
+Step 2: Generate the final WoT TD specification document using the generated WoT resources along with a static ```index.html```
+
+<img title="WoT Toolchain Overview" src="images/toolchain.svg">
 
-The [uv](https://docs.astral.sh/uv/) package manager.
+## Prerequisites
+* Python 3.11 or greater. [Download and install Python.](https://www.python.org/downloads/)
+* The [uv](https://docs.astral.sh/uv/) package manager.
 
 ## Quick Start
 1. Clone the repository and navigate to the project directory:
-```
+```bash
 git clone https://github.com/w3c/wot-thing-description-toolchain-tmp.git
 cd wot-thing-description-toolchain-tmp
 ```
-2. Run the script using `uv`
-```
-uv run main.py -h
+2. Install the package
+```bash
+uv install
 ```
 
 ## Usage
-The main.py script supports various options:
+See the list of all WOTIS commands:
+```bash
+wotis --help
```

Comment:
```suggestion
uv run wotis --help
```

Reply:
I suggest we keep the `wotis --help`, because the project installs it as a standalone command, making it simpler to use.